projects/level-money/index.js

```diff
@@ -37,6 +40,7 @@ module.exports = {
         tvl: sumTokensExport({ 
           owners: [LEVEL_STAKING_CONTRACT, ...LEVEL_RESERVE_MANAGERS, ...LEVEL_WRAPPED_TOKENS], 
           tokens: TOKEN_CONTRACTS,
+          tokensAndOwners: [[LEVEL_USD_USDC_CURVE_LP_TOKEN, LEVEL_USD_POINTS_FARM]]
```

Comment:
we track this under pool2: https://docs.llama.fi/list-your-project/what-to-include-as-tvl

Reply:
fixed!